### ISSUE TITLE
Get an error each time I log into Odoo

### ISSUE THREAD

**Issue by lapagept** [OCA/web#66]
Only web_shortcuts is installed.

```
Odoo Server Error

Traceback (most recent call last):
  File "/home/odoo/openerp/http.py", line 518, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo/openerp/http.py", line 539, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo/openerp/http.py", line 295, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo/openerp/service/model.py", line 113, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo/openerp/http.py", line 292, in checked_call
    return self.endpoint(*a, **kw)
  File "/home/odoo/openerp/http.py", line 755, in __call__
    return self.method(*args, **kw)
  File "/home/odoo/openerp/http.py", line 388, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/addons/web/controllers/main.py", line 949, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/home/odoo/addons/web/controllers/main.py", line 941, in _call_kw
    return getattr(request.registry.get(model), method)(request.cr, request.uid, *args, **kwargs)
AttributeError: 'NoneType' object has no attribute 'get_user_shortcuts'
```

**Comment by bwrsandman** [comment]
I don't know web_shortcuts, but it looks like the model doesn't exist.
This is likely the expression that's failing:

``` python
getattr(request.registry.get(model), method)
# method = 'get_user_shortcuts'
# model = a model which doesn't exist
# request.registry.get(model) = None
```

I'm assuming this is the line which causes the error:

https://github.com/OCA/web/blob/61637c566fe34853fa263eac942e3c8f9cbf113d/web_shortcuts/static/src/js/web_shortcuts.js#L49

Ping @adrienpeiffer

**Comment by matthieuchoplin** [comment]
Hi,

I do not think that the use case given is acurrate enough to reproduce the issue.

I think this is likely what has happened (and what I have reproduced):
- the app web_shortcuts is installed, so the star icon is created to be able to add shortcut
- the app web_shortcuts is uninstalled, the end user expects that the star icon gets removed but it is still there
- the end user click on it the star icon, but because the module has been removed, the object does not exist anymore and this is why we have this error.

So I believe the issue comes from the **uninstallation** of the module, not its installation.
